sdk/ai/azure-ai-projects/samples/evaluation/sample_evaluations_async.py

```diff
@@ -60,20 +62,36 @@ async def main() -> None:
 
             print("Create an evaluation")
             evaluation: Evaluation = Evaluation(
-                display_name="Sample Evaluation",
-                description="Sample evaluation for testing",  # TODO: Can we optional once bug 4115256 is fixed
-                data=InputDataset(id="<dataset_id>"),  # TODO: update this to use the correct id
+                display_name="Sample Evaluation Async",
+                description="Sample evaluation for testing",
+                data=InputDataset(id="azureai://accounts/anksing1rpeastus2/projects/anksing1rpeastus2project/data/eval-data-2025-05-07_165118_UTC/versions/1"),
```

Comment:
Get this from env variable, or write code to get it from project_client.connection.xyz methods

Reply:
There is code in sample but commented out since dataset is not working as of now